docs/pages/kubernetes-access.mdx

```diff
@@ -46,7 +46,7 @@ proxy must have `kube_listen_addr` set and optionally `kube_public_addr` set.
 
```

Comment:
Can you please review this from the docs section? I removed this part from the docs completely

Reply:
I meant all changes to kubernetes-access.mdx, new docs don't reference the script, but it is still helpful to have in examples for some folks who may need it

Reply:
Rebased with your changes and added a new Guide page.
PTAL